src/services/api/admin/donate/bank-details-uah/bank-details-uah-api.ts

```diff
@@ -0,0 +1,31 @@
+import { AxiosInstance } from 'axios';
+import { API_ROUTES } from '../../../../../const/common/api-routes/main-api';
+import { UahBankDetailsType } from '../../../../../types/admin/donate';
+
+export const BankDetailsUahApi = {
+    getAll: async (client: AxiosInstance): Promise<UahBankDetailsType[]> => {
+        const response = await client.get<UahBankDetailsType[]>(API_ROUTES.DONATE.BANK_DETAILS_UAH);
+        return response.data;
+    },
+
+    create: async (client: AxiosInstance, bankDetails: Omit<UahBankDetailsType, 'id'>): Promise<UahBankDetailsType> => {
+        const response = await client.post<UahBankDetailsType>(API_ROUTES.DONATE.BANK_DETAILS_UAH, bankDetails);
+        return response.data;
+    },
+
+    update: async (
+        client: AxiosInstance,
+        id: number,
+        bankDetails: Partial<UahBankDetailsType>,
+    ): Promise<UahBankDetailsType> => {
+        const response = await client.put<UahBankDetailsType>(
```

Comment:
Same question here, should it be `.put` or `.patch`?
```suggestion
        const response = await client.patch<UahBankDetailsType>(
```

Reply:
I agree with you, but on the backend we only have an endpoint with PUT.